src/Symfony/Component/Security/Core/Authentication/Token/TokenInterface.php

```diff
@@ -19,7 +19,9 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  *
- * @method string[] getRoleNames() The associated roles - not implementing it is deprecated since Symfony 4.3
+ * @method array __serialize Returns all the necessary state of the object for serialization purposes - not implementing it is deprecated since Symfony 4.3 and will be required in Symfony 5.0.
+ * @method void     __unserialize(array $data) Restores the object state from an array given by __serialize() - not implementing it is deprecated since Symfony 4.3 and will be required in Symfony 5.0.
+ * @method string[] getRoleNames()                                                                            The associated roles - not implementing it is deprecated since Symfony 4.3
```

Comment:
Indentation looks weird.

Reply:
Indentation was changed by the fabbot patch.

Reply:
But yeah actually it is really weird 😕

Reply:
please make it the nest you can fabbot is just a tool, it doesn't have the last word

Reply:
fixed